src/CADL.Extension/Emitter.Csharp/src/type/InputParameter.ts

```diff
@@ -27,4 +27,5 @@ export interface InputParameter {
     Explode: boolean;
     ArraySerializationDelimiter?: string;
     HeaderCollectionPrefix?: string;
+    SpreadSource?: string;
```

Comment:
Do we need to update the models in the `Input` namespace accordingly to this change?

Reply:
I remove this now. It is for spread model with multiple target.